### ISSUE TITLE
Last row is not rendered when paginationSizeSelector is set to "true" 

### ISSUE THREAD

**Issue by tanuj916** [olifolkerd/tabulator#2924]
**Describe the bug**
To display all data, when `true` is set to `paginationSizeSelector` array, the last row is not rendered. 

**Tabulator Info**
- 4.7.2

**Working Example**
https://codepen.io/tanuj916/pen/oNxgMYR

**To Reproduce**
A step by step guide to recreate the issue in your JS Fiddle or Codepen:
1. Go to https://codepen.io/tanuj916/pen/oNxgMYR
2. Set **Page Size** as **"All"**
3. Scroll to the last row

**Expected behavior**
All the rows should be displayed when **Page Size** is set as **"All"**

**Desktop (please complete the following information):**
 - OS: Mojave
 - Browser Chrome
 - Version 83

**Smartphone (please complete the following information):**

**Example code**
In this example, **paginationSizeSelector** is set as **[4, 10, true]**
When  **Page Size** is set as **10**, all 10 rows are displayed
When  **Page Size** is set as **All**, only 9 rows are displayed

```
var tabledata = [
  {id:1, name:"Oli Bob", age:"12", col:"red", dob:""},
  {id:2, name:"Mary May", age:"1", col:"blue", dob:"14/05/1982"},
  {id:3, name:"Christine Lobowski", age:"42", col:"green", dob:"22/05/1982"},
  {id:4, name:"Brendon Philips", age:"125", col:"orange", dob:"01/08/1980"},
  {id:5, name:"Tanuju", age:"16", col:"yellow", dob:"31/01/1999", details: "test"},
  {id:6, name:"Bruce Wayne", age:"12", col:"red", dob:""},
  {id:7, name:"Clark Kent", age:"1", col:"blue", dob:"14/05/1982"},
  {id:8, name:"Jonn Jonzz", age:"42", col:"green", dob:"22/05/1982"},
  {id:9, name:"Brendon Philips", age:"125", col:"orange", dob:"01/08/1980"},
  {id:10, name:"Barry Allen", age:"16", col:"yellow", dob:"31/01/1999", details: "test"}
];
var table = new Tabulator("#example-table", {
   data:tabledata,
   layout: "fitColumns",
   pagination: "local",
   paginationSize: 2,
   paginationSizeSelector: [4, 10, true],
   columns:[
   {title:"Name", field:"name"},
   {title:"Age", field:"age"},
   {title:"DOB", field:"dob"},    
   {title:"Favourite Color", field:"col"},
   {title: "Details", field: "details"},
   ]
});
```

**Comment by olifolkerd** [comment]
Hey @tanuj916 

Thanks for the headsup, i will look at resolving that in the next release

Cheers

Oli :)

**Comment by olifolkerd** [comment]
Hey @tanuj916 

I have pushed a fix for this to the 4.8 branch that will be released later this month.

Cheers

Oli :)

**Comment by tanuj916** [comment]
@olifolkerd that's great, thanks..

if anyone needs the fix for older versions, please follow this:

set data length as the last array value
 `paginationSizeSelector: [25, 50, 100, tableData.length]`

and in `renderComplete()` update the select
```
            let pageSize = document.querySelector(".tabulator-page-size");
            let showALl = pageSize.options[pageSize.options.length - 1];
            showALl.innerHTML = "Show All";
```
